adapter/src/main/java/ar/edu/undec/adapter/service/rest/RegistrarCursoController.java

```diff
@@ -0,0 +1,27 @@
+package ar.edu.undec.adapter.service.rest;
+
+import ar.edu.undec.adapter.service.domain.CursoDTO;
+import curso.input.RegistrarCursoInput;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping("curso")
```

Comment:
Creo que acá faltaría la barra "/", pero no estoy plenamente seguro. Veremos cuando lo levantes e intentes pegar a 
```http
POST http://localhost:8080/curso
```

```suggestion
@RequestMapping("/curso")
```